### ISSUE TITLE
Status bit for overloaded streams

### ISSUE THREAD

**Issue by BigEvilCorporation** [sikthehedgehog/Echo#18]
Currently a busy stream will just cause slow playback, it would be handy to know if this happens via a status bit/error code for debugging and QA

**Comment by marcb0t** [comment]
Yes, I've noticed some slow down/speedups with a stream of pitchbend commands for FM tom toms. Maybe this will resolve the issue?

**Comment by sikthehedgehog** [comment]
That won't solve the issue, the only thing that will solve the issue is making Echo faster :P (ironically, if I do the 2ch PCM thing, it'll get worse, not better :/ but every composer asks me if it's there probably because they heard Ristar does it so they assume everybody can do it)
